app/controllers/Application.java

```diff
@@ -3,10 +3,12 @@
 import play.mvc.Controller;
 import play.mvc.Result;
 
+@SuppressWarnings("javadoc")
```

Comment:
Ignoring missing Javadocs is fine for test classes, but for production code the reason for the custom compiler check is to actually ensure all public API is documented, so we should add documentation here.